packages/compass-smoke-tests/src/cli.ts

```diff
@@ -145,6 +145,60 @@ async function getTestSubject(
   }
 }
 
+type Arch = 'arm64' | 'x64';
+
+type PlatformShortName =
+  | 'darwin-arm64'
+  | 'darwin-x64'
+  | 'windows'
+  | 'linux_deb'
+  | 'linux_rpm';
+
+function getPlatformShortName(
```

Comment:
The URL for downloading the latest release ends in things like osx, darwin-arm64, darwin-x64, windows, linux_dev or linux_rpm. This just calculates that bit. I'm calling it a "platform short name" until we think of something better.